src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -811,6 +811,8 @@ define ReportOpioidMedicationRequests:
     Name:  ConceptText(O.medication as FHIR.CodeableConcept),
     Start: DateTimeText(O.authoredOn),
     End:   null as String, // MedicationRequest end date not supported in R4,
+    //Amount: QuantityText(O.value as FHIR.Quantity),
+    Quantity: O.dispenseRequest.quantity,
```

Comment:
is it perhaps `O.dispenseRequest.quantity.value`? I was looking at the FHIR json `  "dispenseRequest": {
          "extension": [
            {
              "url": "http://cosri.org/fhir/pharmacy_name",
              "valueString": "REAL CHAIN STORE 1 (CO)"
            },
            {
              "url": "http://cosri.org/fhir/last_fill",
              "valueDate": "2020-03-28"
            }
          ],
          "quantity": {
            "value": 30
          },
          "expectedSupplyDuration": {
            "value": 8,
            "unit": "days",
            "system": "http://unitsofmeasure.org",
            "code": "d"
          }
        }`

Reply:
I feel like I tried that first and got a processing error, but I can try it again. It does seem more correct.

Reply:
@mldorr Maggie, I tried this out in my dev instance, I think this is correct:  **`Quantity: O.dispenseRequest.quantity.value.value`** note the additional, `.value`
![Screen Shot 2020-08-11 at 9 40 12 AM](https://user-images.githubusercontent.com/12942714/89924468-e8885100-dbb6-11ea-96b7-d83722a598e7.png)